vhdbuilder/packer/components.json

```diff
@@ -144,7 +144,8 @@
         "ciprod06112021",
         "ciprod05202021-hotfix",
         "ciprod06112021-1",
-        "ciprod08052021"
+        "ciprod08052021",
```

Comment:
Is it ok to have this older version? And can that be removed since it will not be used?

Reply:
yes, ciprod08052021 ok to have and we will remove old ones next agent release.

Reply:
The older version will still be used. We only plan to use the newer version if there is an issue toggling a few customers to the new version